querydsl-jpa/src/main/java/com/querydsl/jpa/NativeSQLSerializer.java

```diff
@@ -206,7 +206,7 @@ public void visitConstant(Object constant) {
     @Override
     protected void visitOperation(Class<?> type, Operator operator, List<? extends Expression<?>> args) {
         if (operator == SQLOps.ALL
-                && !RelationalPath.class.isInstance(args.get(0))
+                && !(args.get(0) instanceof RelationalPath)
```

Comment:
Why not leave it alone? Just introducing risk into a class that doesn't need touched for personal preference.

Reply:
Resolved an Intellij warning. I can revert it if you think it's risky.

Reply:
Ah OK, keep it in, I naively assumed you just preferred the new syntax.